docs/explore/introduction/first-party.md

```diff
@@ -97,3 +97,88 @@ architecture.
 <!--Third-party oracles are both insecure and expensive (see the
 <a href="/api3-whitepaper-v1.0.3.pdf#page=10" target="_blank">API3
 Whitepaper</a><externalLinkImage/> _Issues with Third-Party Oracles as Middlemen_ for a detailed explanation). In contrast, first-party oracles are both secure and cost-efficient due to not having a middleman on the interface path. -->
+
+## First-party oracle governance 
+
+### Legacy oracle governance 
+
+Traditionally an oracle network makes the same request to multiple
+independent oracles and reduces their responses to a single answer through
+predetermined consensus rules. This is implemented as a smart contract called the
+aggregator. 
+
+Individual malicious oracles cannot manipulate the outcome of this
+process, which provides a degree of decentralization and trustlessness. Here, an important thing to consider is how the oracle network is governed. 
+
+
+> <img src="../assets/images/central-governance.png" width="300"/> > <img src="../assets/images/decentral-governance.png" width="300"/>
+
+<!--::: danger TODO
+
+The diagram above and below do not illustrate the governance issue well. They
+need to be blended into one. Or maybe they go away.
+
+::: -->
+
+If a central entity can switch the oracles or APIs used in the aggregator in and out,
+or even replace the aggregator itself making use of a proxy mechanism, they can
+effectively manipulate the oracle network output at will. 
+
+This eliminates the
+decentralization and trustlessness qualities that using an oracle network
+provides. Therefore, it is not adequate to use an oracle network to achieve
+decentralization, this oracle network must be governed decentrally as well.
+
+::: Info
+
+See the medium article,
+[Why API3 DAO?<externalLinkImage/>](https://medium.com/api3/why-api3-dao-not-api3-corp-2dde51c537c1)
+on DAOs and decentralized governance.
+
+:::
+
+### Decentrally governed oracles
+
+Decentrally-governed networks of first-party oracles would adquately solve the API connectivity problem. Consider the following:
+
+1. Decentralized applications need access to APIs.
+2. It is optimal to interface APIs to smart contract platforms through first-party
+  oracles.
+3. For API level decentralization, decentrally-governed oracle networks should be
+  employed.
+
+
+  > <img src="../assets/images/dapi.png" width="350"/>
+
+
+API3 delivers decentralized interfaces to APIs to facilitate this through governance.
+
+A decentralized API is a product which contains
+
+1. APIs interacting with oracle networks decentralized interfaces
+2. Decentralized governance and interfaces
+
+By taking advantage of the advantages offered by decentralization, developers can build applications with elevated security and trustworthiness.
+
+
+<!-->
```

Comment:
I happened to see the build error is confusing with regards to the line number. Turns out it can be fixed by removing the trailing `>` character here.

Reply:
You mean the <!--> as an illegal comment?

This PR is not getting merged anyway but it is interesting to understand more about it. Thanks

Reply:
Yep that's what I meant and thanks for the clarification